CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## Version 0.1 (Release Date: 2024-05-09)
+
+### New Features
```

Comment:
It's not really New Features. 
Unless you need to follow some official format, I would say "initial release" (Or "initial features"